pkg/azurefile/utils.go

```diff
@@ -265,3 +275,47 @@ func replaceWithMap(str string, m map[string]string) string {
 	}
 	return str
 }
+
+func getAzcopyJob(dstFileshare string) (AzcopyJobState, string, error) {
+	cmdStr := fmt.Sprintf("azcopy jobs list | grep %s -B 3", dstFileshare)
+	out, err := exec.Command("sh", "-c", cmdStr).CombinedOutput()
+	if err != nil && err.Error() != "exit status 1" {
+		return AzcopyJobError, "", fmt.Errorf("couldn't list jobs in azcopy %v", err)
+	}
+	jobid := ""
+	jobSegments := strings.Split(string(out), "JobId: ")
+	if len(jobSegments) < 2 {
+		return AzcopyJobNotFound, "", nil
+	}
+	jobSegments = jobSegments[1:]
```

Comment:
btw, for better test coverage, I think you could split this `getAzcopyJob` into at least two functions, the second function only parses the output of `azcopy jobs list | grep %s -B 3`, by that way, you could simulate the output, and it's easier to write ut for this function

Reply:
done, split two func `parseAzcopyJobList` and `parseAzcopyJobShow` and add ut